djangocms_frontend/helpers.py

```diff
@@ -133,7 +133,13 @@ def delete_plugin(plugin):
 
 
 def coerce_decimal(value):
+    """Force value to be converted to decimal.Decimal or return None"""
     try:
         return decimal.Decimal(value)
     except TypeError:
         return None
+
+
+def export(*args):
+    """Dummy function to avoid linters to complain about unused imports"""
```

Comment:
Can we not just `# noqa` it?

Reply:
It seems lgtm.com ignores the `# noqa`. But then again lgtm.com is not the world rule. (Also, since I've changed some of the ways framework dependent imports are handled, most of the complaints will go away anyway.)